adoc/admin-updates.adoc

```diff
@@ -265,7 +265,13 @@ with the `kured` reboot daemon.
 === Disabling Automatic Updates
 
 Nodes added to a cluster have the service `skuba-update.timer`, which is responsible for running automatic updates, activated by default.
+
 This service calls the `skuba-update` utility and it can be configured with the `/etc/sysconfig/skuba-update` file.
+[NOTE]
+===
+`skuba-update` uses the flags `--non-interactive --non-interactive-include-reboot-patches`. `--non-interactive-include-reboot-patches`, when used in conjunction with `--non-interactive`, will cause patches with the `rebootSuggested-flag` not to be skipped. The reboot will be managed by `kured`.
```

Comment:
```suggestion
`skuba-update` uses the flags `--non-interactive` and `--non-interactive-include-reboot-patches`. The `--non-interactive` flag causes zypper to use default answers to questions rather than prompting a user.  In non-interactive mode, the `--non-interactive-include-reboot-patches` flag causes patches with the `rebootSuggested-flag` to not be skipped. Zypper does not perform the reboot directly. Instead, `kured` will be used to safely schedule reboots as needed.
```

Minor rearrangement to avoid starting a sentence with a hyphen, and some similar grammatical tweaks.

Reply:
Looks ok, thanks.